packs/hubot/actions/workflows/post_result.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  chain:
+    -
+      name: "format_result"
```

Comment:
I thought the main motivation here was to do no formatting in python and move it all to hubot. I feel we should simply post the entire execution or payload of the notification.